### DISCUSSION TITLE
Is there a way to use pretrained acoustic models?

### DISCUSSION THREAD

**Discussion by haru0l** [openvpi/DiffSinger#84] [Q&A] [answered]
As per title, I would like to know if there is a way for this as it would speed up training time. 🙏

**Comment by yqzhishen** [comment]
If you are talking about any universal pretrained models, the answer is no. Universal pretrained models aren't that useful, because they are bound to dictionaries, and you cannot modify most of the config keys, either.

You can reuse your former checkpoints as your private pretrained models. But you cannot change the dictionary or add/remove most of its functionalities. To use it, increase `max_updates` in your config, replace the learning rate with a larger value (as it represents the initial lr), and resume training.

What do you expect that pretrained models can do?

**Comment by yqzhishen** [comment, marked as answer]
Finetuning will be supported in #108. It will soon be merged and will be formally released in v2.1.0.
